Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ipc-channel"
-version = "0.18.0"
+version = "0.19.0"
```

Comment:
My read of the uses of mio in this codebase is that none of its types are exposed in ipc-channel's public API. We should be able to release 0.18.1 instead.

Reply:
Makes sense to me! I've gone ahead and done that.